iaso/api/org_unit_change_requests/filters.py

```diff
@@ -22,6 +22,12 @@ class OrgUnitChangeRequestListFilter(django_filters.rest_framework.FilterSet):
     parent_id = django_filters.NumberFilter(method="filter_parent_id", label=_("Parent ID"))
     groups = django_filters.CharFilter(method="filter_groups", label=_("Groups IDs (comma-separated)"))
     project = django_filters.NumberFilter(field_name="org_unit__org_unit_type__projects", label=_("Project ID"))
+    created_date_from = django_filters.DateFilter(
+        lookup_expr="gte", field_name="created_at__date", input_formats=["%d-%m-%Y"]
+    )
+    created_date_to = django_filters.DateFilter(
+        lookup_expr="lte", field_name="created_at__date", input_formats=["%d-%m-%Y"]
+    )
```

Comment:
1. I did the same thing in Polio
   - you can also use a `DateFromToRangeFilter()` if you want only one field in the filter
   - **but** you can also use your solution because it works too, it's up to you 😄 
2. but use `settings.API_DATE_INPUT_FORMATS` instead of hardcoding the format `"%d-%m-%Y"`

---

Just in case you want to see the `DateFromToRangeFilter()`:

- [backend code is here](https://github.com/BLSQ/iaso/blob/c9f54413318f9af96ecb390b1743d6a1ff2b1503/plugins/polio/api/notifications/filters.py#L22-L27)
- [frontend](https://github.com/BLSQ/iaso/pull/941/files#diff-48f4faaff58cec093a0f5cde5212493e1e909fbfb341a8e382c03aa3723c712eR10-R11) (with `_after` and `_before`)